cypress/support/index.js

```diff
@@ -75,3 +78,27 @@ if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
     cy.stopDummyServer();
   });
 }
+
+/**
+ * Make setup step in here so that all the test with MDS files in dashboards-assistant
+ * won't need to call these commands.
+ */
+if (
+  Cypress.env('DASHBOARDS_ASSISTANT_ENABLED') &&
+  Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')
+) {
+  const originalBackendEndpoint = currentBackendEndpoint.get();
+  before(() => {
+    currentBackendEndpoint.set(currentBackendEndpoint.REMOTE_NO_AUTH);
+    cy.addAssistantRequiredSettings();
+    cy.readOrRegisterRootAgent();
+    currentBackendEndpoint.set(originalBackendEndpoint, false);
+    cy.startDummyServer();
+  });
+  after(() => {
+    currentBackendEndpoint.set(currentBackendEndpoint.REMOTE_NO_AUTH);
```

Comment:
Prefer move get original before set. The original endpoint may be changed in the final after right?

Reply:
Sure, the code has been updated.